### DISCUSSION TITLE
About device detection and same/other device

### DISCUSSION THREAD

**Discussion by henkealg** [ActiveLogin/ActiveLogin.Authentication#413] [Q&A] [answered]
Thank you for this very useful project. 👍 

From [the details in this issue](https://github.com/ActiveLogin/ActiveLogin.Authentication/issues/390), I presume we will need to present both the "other device" and "same device" options for the user when using the latest version (6.0.3), since the device detection and redirect done in the current test.bankid.com example is not implemented yet. 

Is this correct? 

I guest one intermediate fix would be to add a link to the "same device" option from the "other device" QR-code view to make use of one entry point for auth from our login view.

**Comment by PeterOrneholm** [comment, marked as answer]
Hi Henrik and thank you!

> I guest one intermediate fix would be to add a link to the "same device" option from the "other device" QR-code view to make use of one entry point for auth from our login view.

Yes, this is how I would implement it, override the built in template and link to the "other" scheme.

Hopefully we will build this into the product in the future.
